logicrunner/builtin/contract/member/member.go

```diff
@@ -288,6 +290,11 @@ func (m *Member) transferCall(params map[string]interface{}) (interface{}, error
 		return nil, fmt.Errorf("incorect input: failed to get 'amount' param")
 	}
 
+	asset, ok := params["asset"].(string)
```

Comment:
do we REALLY need this for now?

Reply:
You can talk about this with Vladimir ;)